src/main/resources/static/img/logo-cu-white-outlined.svg

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
lol i see you stole carleton's svg logo 😉

Reply:
lol yup :)